.github/workflows/release.yaml

```diff
@@ -42,7 +40,7 @@ jobs:
       - name: Release
         env:
           NPM_CONFIG_USERCONFIG: /dev/null
-          GITHUB_TOKEN: ${{ env.GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
You can just delete this line

Reply:
It was there previously so would like to keep it to be safe, in case it's not picked up in the env! Think it still may be needed

Reply:
Okay, fair enough